Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -57,34 +54,34 @@ public async Task UpdateContactPoliciesSettingsAsync(string accountName, Entitie
             const string updateQuery =
                 @"update [UserShippingLimit] set Active = @Active, Interval = @IntervalInDays, Amount = @EmailsAmountByInterval
                 from [UserShippingLimit] usl
-                inner join [User] u on u.IdUser = usl.IdUser where u.Email = @Email and usl.Enabled = 1;";
+                inner join [User] u on u.IdUser = usl.IdUser where u.IdUser = @IdUser and usl.Enabled = 1;";
```

Comment:
Now you can remove the join with `User` table, right?

```diff
- inner join [User] u on u.IdUser = usl.IdUser where u.IdUser = @IdUser and usl.Enabled = 1;";
+ where usl.IdUser = @IdUser and usl.Enabled = 1;";
```

Reply:
👍